src/components/Button/README.md

```diff
@@ -48,7 +48,11 @@ In order to override the theme, use Wix Stylable extends capabilities and wrap w
 2. Create a component that uses it
     ``` javascript
     import Button from 'wix-ui-tpa/Button';
-    import buttonStylesExt from './ButtonExt.st.css';
+    import { st, classes } from './ButtonExt.st.css';
 
-    const ButtonExt = (props) => <Button {...props} {...buttonStylesExt('root', {}, props)}/>;
+    const ButtonExt = (props) => 
+        (<Button 
+            className={st(classes.root)} 
+            {...props} 
```

Comment:
```suggestion
            {...props} 
            className={st(classes.root, props.className)} 
```

Reply:
This is just an example of using the component, so most consumers won't accept className as a prop, they would just use it like this

Reply:
i understand this (and some others) are just examples, but it would be good to show proper usage in examples that can avoid potential missuses.

it's easy for us, cause we know how it works, but for someone new it can be not that trivial